README.md

```diff
@@ -29,14 +29,23 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
+Map utilizes return values and returns a new array of the same size while forEach calls a provided function on each element in your array.
 
 2. What is the difference between a function and a method?
+Methods have a reciever and functions do not.
 
 3. What is closure?
+Closure is the combination of a function and the lexical environment within which that function was declared.
```

Comment:
Here's my answer:
`Closure is a type of scope. If the compiler can't find a variable it's looking for on the local scope it "looks up" to the scope above it. The way JS passes variables down through scope is called closure.`